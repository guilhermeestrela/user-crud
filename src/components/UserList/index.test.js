import { render } from '@testing-library/react'
import '@testing-library/jest-dom'

import { UserList } from './'

describe('UserList', () => {
    it('Should render UserList correctly', () => {
        const usersMock = [
            {
                id: 1637098364443,
                name: "Test User",
            }
        ]
        const { container, getByText } = render(<UserList users={usersMock} />)

        expect(container).toMatchSnapshot()
        expect(getByText('Test User')).toBeInTheDocument()
    })
})
