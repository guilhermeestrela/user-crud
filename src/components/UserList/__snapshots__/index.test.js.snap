// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserList Should render UserList correctly 1`] = `
<div>
  <ul
    class="user-list"
  >
    <li
      class="user-item"
    >
      <p
        class="user-name"
      >
        Test User
      </p>
      <button
        class="secondary-button"
      >
        Remove
      </button>
    </li>
  </ul>
</div>
`;
